libcontainer/cgroups/fs2/memory.go

```diff
@@ -138,6 +143,7 @@ func getMemoryDataV2(path, name string) (cgroups.MemoryData, error) {
 	}
 	usage := moduleName + ".current"
```

Comment:
WDYT of adopting a similar pattern to https://github.com/opencontainers/runc/pull/4038/files#diff-597c9eb164fc8d34c7fa865544c067ee504c4fd9bf745b55e9dc4810881d9035R139-R174 ? in particular, I think collecting memory.peak should be best effort and if it's not present, some metrics should still be returned

Reply:
Well, if no `memory.peak` file is present, it will be a ENOENT error and https://github.com/opencontainers/runc/pull/4010/files#diff-597c9eb164fc8d34c7fa865544c067ee504c4fd9bf745b55e9dc4810881d9035R169 this place will not make an error return. Due to the logic of function `fscommon.GetCgroupParamUint`, `value` will be `0` as expected.